docs/fits_region.rst

```diff
@@ -0,0 +1,89 @@
+.. _gs-fits:
+
+Reading/writing to FITS region files
+====================================
+
+The regions package provides the functionality to serialise and de-serialise
+Python lists of `~regions.Region` objects to FITS region file. It is a FITS
+Region Binary Table that defines a spatial region of a two-dimensional image in
+pixels. The name of the header must be ``REGION`` for the
+`~regions.read_fits_region` to parse the table. The file is read using the
+`~astropy.table.Table.read` method which gets converted into an
+`~astropy.table.Table` object. This table is then parsed using
+`~regions.FITSRegionParser`. It converts the FITS region table to
+`~regions.ShapeList` object, which is a list of `~regions.Shape` each
+representing one FITS region. The `~regions.Shape` objects can be converted to
+`~regions.Region` objects. Some of the regions such as ``rectangle``,
+``rotrectangle``, ``pie``, ``sector`` are not supported by this
+package.
+
+.. code-block:: python
+
+    >>> from regions import FITSRegionParser
+    >>> from astropy.table import Table
+    >>> from astropy.utils.data import get_pkg_data_filename
+    >>> filename = get_pkg_data_filename('data/regions.fits',
+    ...                                  package='regions.io.fits.tests')
+    >>> table = Table.read(filename)
```

Comment:
This method will read the first HDU table in the file whether it is a region table or not? On the other hand the `read_fits_region` function loops over all the tables in the FITS file that are identified as regions. Perhaps this distinction could be explained in the documentation.